ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -170,8 +170,7 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
     }
 
     try {
-      const result = await institutionApi().checkEmail(institutionShortName, contactEmail,
-        {signal: this.aborter.signal});
+      const result = await institutionApi().checkEmail(institutionShortName, {value: contactEmail}, {signal: this.aborter.signal});
```

Comment:
If we find a way to make `type: string` work in Swagger, we wouldn't need to change UI code at all.